src/streamlink/plugin/plugin.py

```diff
@@ -189,6 +190,10 @@ def set_option(cls, key, value):
     def get_option(cls, key):
```

Comment:
when the argument is called `b-netloc`

`-` won't work in get_option `_` must be used,
well thats **backport compatibility**

but shouldn't it work with `-` now too or should `self.get_argument` be used now



```
self.get_option("b-netloc") == None
self.get_option("b_netloc") == ['test.com', 'test.test']
```

Reply:
I guess the only caveat here is that the keys `b_netloc` and `b-netloc` will be equivalent - I don't think this is a problem though.

Reply:
The other option is to use the `dest` keyword argument to override the option name.